Sources/Baggage/BaggageContext.swift

```diff
@@ -60,10 +60,9 @@ public struct BaggageContext {
         }
     }
 
-    public var baggageItems: [AnyBaggageContextKey: Any] {
-        // TODO: key may not be unique
-        self._storage.reduce(into: [:]) {
-            $0[$1.key] = $1.value.value
+    public func forEach(_ callback: (AnyBaggageContextKey, Any) -> Void) {
```

Comment:
In line with the `forEach`s in `swift-tracing` 👍

Reply:
Yeah, and enforced to match via the `BaggageContextProtocol`